src/com/esotericsoftware/kryo/util/MapReferenceResolver.java

```diff
@@ -30,18 +30,21 @@
  * for get or put.
  * @author Nathan Sweet */
 public class MapReferenceResolver implements ReferenceResolver {
-	private static final int CAPACITY = 2048;
-	private static final int MAXIMUM_CAPACITY = 8192;
+	private static final int DEFAULT_CAPACITY = 2048;
 
 	protected Kryo kryo;
 	protected final IdentityObjectIntMap<Object> writtenObjects = new IdentityObjectIntMap<>();
 	protected final ArrayList<Object> readObjects = new ArrayList<>();
 	private final int maximumCapacity;
 
+	/** Creates a reference resolver with a default maximum capacity of 2048 */
 	public MapReferenceResolver () {
-		this(MAXIMUM_CAPACITY);
+		this(DEFAULT_CAPACITY);

```

Comment:
This change affects the code from line 90 to 93, which is more probable to kick in.
As I understand it things would be faster with the higher max capacity. So why not use a higher default (8k)?

Reply:
> This change affects the code from line 90 to 93, which is more probable to kick in.

Correct. But the change was merged relatively recently, so adjusting this should not affect a lot of users. And they can always increase the `maxCapacity` to 8k if the wish.

> So why not use a higher default (8k)?

We could change it to 8k, but `2048` is the default for `DefaultClassResolver` as well and I wanted to keep it consistent.

Reply:
Ok, thanks!